docs/_blog/liana-payjoin.md

```diff
@@ -0,0 +1,44 @@
+
+## Privacy Leak Explained
+
+Inheritance planning is a crucial yet often overlooked part of Bitcoin self-custody. While Bitcoin offers unmatched financial sovereignty, it introduces [unique challenges](https://wizardsardine.com/blog/the-state-of-bitcoin-inheritance-planning/) when it comes to passing on funds to future generations. Liana addresses this with inheritance policies that activate only after a timelock. For example, a user might control a primary key for daily use, while a recovery key held by a beneficiary is time locked for future access.
+
+However, there's a catch: most users don't frequently spend their Bitcoin. [A 2024 study](https://www.sciencedirect.com/science/article/pii/S0378437124008045) shows the age of unspent coins follows a power-law distribution. In other words, coins are either spent quickly or left untouched for long periods. [Data also suggests](https://charts.bitbo.io/hodl-waves/) that most coins remain unspent for months or years. To avoid unintentionally triggering a recovery path, users often resort to self-spending or reconciliation to reset timelocks.
+
+<img alt="Liana time locked coins" src="../assets/liana-coins.png" />
+
+But this reconciliation comes at a cost to privacy. These transactions create an identifiable on-chain footprint, making it easier for on-chain surveillance to estimate a user's balance and transaction history. This is primarily due to the [common input ownership heuristic](https://en.bitcoin.it/wiki/Common-input-ownership_heuristic), which assumes all inputs in a transaction belong to the same entity. Combined with unique transaction patterns that act like fingerprints, these heuristics allow on-chain analysts to cluster a user's addresses and glean sensitive information.
+
+## Payjoin as a Privacy-Preserving Refresh
+
+This is where Payjoin shines. It allows Liana users to refresh near-expiry coins during normal payments. For example, when a user receives a payment via Payjoin, they can opportunistically include expiring coins to refresh their timelocks. This same logic can apply to self-spends—users can initiate a Payjoin when sending a transaction, embedding refresh into routine activity.
+
+Whether as sender or receiver, users benefit from lower fees, improved privacy, and preservation of their inheritance policies.
+
+<img alt="Liana Payjoin" src="../assets/liana-batch-payment.svg" />
+
+## Caveats
+
+It’s worth noting that the common-input heuristic is not defeated when using Payjoin with P2WSH inputs—unless both participants share identical or structurally similar descriptors. This is because the spending script is revealed on-chain, allowing fingerprinting based on script templates.
+
+However, Taproot inputs do not suffer from this issue. With Taproot’s key-path spending, the recovery script remains hidden unless explicitly revealed. This makes it possible to conduct Payjoin transactions between users with different wallet policies while breaking the common-input heuristic.
```

Comment:
> With Taproot’s key-path spending

Generally prefer links to any technical jargon so the reader can learn more if they'd like. I'm usually opposed to assuming the reader knows what we mean beyond the basics. If it's not overly pedantic, I either like to spell it out or provide links to learn more.

Reply:
Very good point!